File: 2998-count-symmetric-integers/count-symmetric-integers.cpp
class Solution {
public:
bool symmetric(int n) {
    vector<int> sam; int sum1=0,sum2=0 ;
    while(n>0){sam.push_back(n%10); 
         sum1=sum1+n%10; n/=10;}
    if(sam.size()%2!=0){return false;}
     for(int i=0;i<sam.size()/2;i++){sum2=sum2+sam[i]; }
           return sum1-sum2==sum2;
    
}
int countSymmetricIntegers(int low, int high) {
    int counter=0;
       for(int i=low;i<=high;i++){
           if(symmetric(i)){counter++;}
    } return counter;
    }

};

File: 2998-count-symmetric-integers/count-symmetric-integers.py
class Solution:
    def countSymmetricIntegers(self, low: int, high: int) -> int:
        count=0
        for num in range(low,high+1):
            temp=[int(digits ) for digits in str(num)]
            if(len(temp)%2!=0):
                continue
            mid=len(temp)//2 
            if(sum(temp[:mid])==sum(temp[mid:])):
                count+=1
        return count